neonvm/tools/vm-builder/main.go

```diff
@@ -369,30 +369,36 @@ func printReader(reader io.ReadCloser) error {
 	return nil
 }
 
-func AddToTar(tw *tar.Writer, filename string, buf bytes.Buffer) error {
+func AddTemplatedFileToTar(tw *tar.Writer, buf *bytes.Buffer, tmplData any, filename string, tmplString string) error {
```

Comment:
I just had deja vu, but I suppose you did too. Is there any reason not to pull things into a util (perhaps not now, and I'm happy to use my fingers to do this.